.github/workflows/updates-e2e.yml

```diff
@@ -0,0 +1,82 @@
+name: Updates e2e
+
+on:
+  workflow_dispatch: {}
+  pull_request:
+    paths:
+      - .github/workflows/updates-e2e.yml
+      - packages/expo-json-utils/**
+      - packages/expo-manifests/**
+      - packages/expo-modules-core/**
+      - packages/expo-structured-headers/**
+      - packages/expo-updates-interface/**
+      - packages/expo-updates/**
+      - packages/expo/**
+  push:
+    branches: [main, 'sdk-*']
+    paths:
+      - .github/workflows/updates-e2e.yml
+      - packages/expo-json-utils/**
+      - packages/expo-manifests/**
+      - packages/expo-modules-core/**
+      - packages/expo-structured-headers/**
+      - packages/expo-updates-interface/**
+      - packages/expo-updates/**
+      - packages/expo/**
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  android:
+    runs-on: macos-11
```

Comment:
macOS runners are slower to provision and could get very expensive to run -- run everything on Linux when possible, like Android tests

Reply:
These tests require an android emulator, and the GHA we have been using for this in all other jobs that use an emulator is recommended to be run on macos: https://github.com/ReactiveCircus/android-emulator-runner#usage

Reply:
I see. Makes sense but could become a big problem one day. Obviously not specific to Expo and affects every Android developer looking to test in CI but isn't great if we can't use Linux. Maybe the answer is to move to Linux VMs that run tests outside of containers.